server_environment/__manifest__.py

```diff
@@ -21,7 +21,7 @@
 
 {
     "name": "server configuration environment files",
-    "version": "9.0.1.2.0",
+    "version": "10.0.1.2.0",
```

Comment:
I personally think maintainers should have the choice in this matter. If one maintainer whishes to maintain feature parity of an addon across Odoo versions, it makes sense to use the version number to express the feature level.